### PR TITLE
feat: Allow to set any value after `FORMAT TYPE` for INSERT command in ClickHouse

### PR DESCRIPTION
**Pull request by viladimiru**

### CODE REVIEW

File: src/autocomplete/databases/clickhouse/clickhouse-autocomplete.ts [hunk 1]
```diff
@@ -98,6 +98,7 @@ const rulesToVisit = new Set([
     ClickHouseParser.RULE_columnIdentifier,
     ClickHouseParser.RULE_identifierOrNull,
     ClickHouseParser.RULE_literal,
+    ClickHouseParser.RULE_any,
```
Comment (viladimiru) [inline src/autocomplete/databases/clickhouse/clickhouse-autocomplete.ts:101]: We need to include rule in `rulesToVisit` to disable keywords suggestion

File: src/autocomplete/databases/clickhouse/grammar/ClickHouseParser.g4 [hunk 2]
```diff
@@ -998,9 +997,49 @@ insertFormatType
...
+    | FORMAT? valuesStatement
+    | selectUnionStatement SEMICOLON? EOF
+    ;
+
+any
+    : identifier
+    | literal
+    | specialSymbol
+    | UNKNOWN_CHAR
+    ;
```
Comment (NikitaShkaruba) [inline src/autocomplete/databases/clickhouse/grammar/ClickHouseParser.g4:1010]: Why not just `.*`?

**Review by NikitaShkaruba — APPROVED** [review]

File: src/autocomplete/databases/clickhouse/grammar/ClickHouseLexer.g4 [hunk 3]
```diff
@@ -444,6 +444,8 @@ fragment OCT_DIGIT : [0-7];
 fragment DEC_DIGIT : [0-9];
 fragment HEX_DIGIT : [0-9a-fA-F];
 
+// Special symbols
+// Don't forget to add all new symbols to the parser rule "specialSymbol"
```
Comment (roberthovsepyan) [inline src/autocomplete/databases/clickhouse/grammar/ClickHouseLexer.g4:448]: do we still need this comment?

File: src/autocomplete/databases/clickhouse/grammar/ClickHouseParser.g4 [hunk 4]
```diff
@@ -998,9 +997,13 @@ insertFormatType
...
 dataClause
-    : FORMAT insertFormatType identifier+ # DataClauseFormat
-    | FORMAT? valuesStatement             # DataClauseValues
-    | selectUnionStatement SEMICOLON? EOF # DataClauseSelect
+    : FORMAT insertFormatTypeExceptValues any
+    | FORMAT? valuesStatement
+    | selectUnionStatement SEMICOLON? EOF
+    ;
+
+any
```
Comment (roberthovsepyan) [inline src/autocomplete/databases/clickhouse/grammar/ClickHouseParser.g4:1005]: nit: I kind of don't like this name
maybe better to call it `anyValue` or `anyToken`?

**Review by roberthovsepyan — APPROVED** [review]

**Review by roberthovsepyan — APPROVED** [review]

**Review by roberthovsepyan — APPROVED** [review]

**Pull request merged**